eng/pipelines/templates/steps/build-artifacts.yml

```diff
@@ -70,6 +70,10 @@ steps:
       twine check $(Build.ArtifactStagingDirectory)/**/*.zip
     displayName: 'Verify Readme'
 
+  - template: /eng/pipelines/templates/steps/run_apistub.yml
```

Comment:
I myself can't see why this APIStubGen step is failing. I'm moving it up above GenerateDocs just in case we're downloading a wrong version of `pylint` or the like earlier in the pipeline.

Reply:
[No luck](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1622691&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=76d9b154-b861-59e4-d8e3-dfd06fc0fc10)